nifc_wildfires/__init__.py

```diff
@@ -0,0 +1,27 @@
+#! /usr/bin/env python
+# -*- coding: utf-8 -*-
+import requests
+
+
+def get_current_perimeters():
+    """
+    Get geojson fire perimeters from new 2021 feed.
+
+    Returns a GeoJSON object.
+    """
+    r = requests.get("https://opendata.arcgis.com/datasets/2191f997056547bd9dc530ab9866ab61_0.geojson")
+    if r.status_code != 200:
+        raise Exception(f"Request for data failed with {r.status_code} status code")
+    return r.json()
+
+
+def get_nifc_incidents():
```

Comment:
If we're renaming the library nifc, I think we can skip calling the functions nifc too. What if this was just `get_incidents` for simplicity's sake?

Reply:
yup